### PR TITLE
Tracing requests without logging results 

### PR DESCRIPTION
**Pull request by frances3006**
- added on onFieldResolveFinish to allow users to define their own behaviour when the request has finished
- added on onFieldResolve to allow users to add extra information to the span such as tags based on the headers or other information from the context

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -140,17 +162,25 @@ export default class OpentracingExtension<TContext extends SpanContext>
...
     return (error: Error | null, result: any) => {
-      if (error) {
-        span.log({ error: JSON.stringify(error) });
-      }
-      if (result) {
-        span.log({ result: JSON.stringify(result) });
+      if (this.onFieldResolveFinish) {
+        this.onFieldResolveFinish(error, result, span);
+      } else {
+        if (error) {
```
Comment (DanielMSchmidt) [inline src/index.ts]: Let's remove the default stuff here, this can be done manually thanks to your changes :)

Comment (frances3006) [reply src/index.ts]: thank you for the quick feedback. I just changed it

**Comment by DanielMSchmidt** [comment]
Thank you for the adjustment, the code looks great now :)

This project deploys itself via semantic-release, therefore we need the [commit messages in the angular commit style](https://gist.github.com/stephenparish/9941e89d80e2bc58a153#format-of-the-commit-message). Thank you for your patience 👍

**Comment by frances3006** [comment]
I have changed the commit message. Let me know if there's anything else to be done. Thank you

**Comment by DanielMSchmidt** [comment]
Awesome, thank you for taking the time :)

**Pull request merged**